docs/validators/sigstore_cosign.md

```diff
@@ -92,8 +92,15 @@ kubectl run altsigned --image=docker.io/securesystemsengineering/testimage:co-si
 | `name` | - | :heavy_check_mark: | See [basics](../basics.md#validators). |
 | `type` | - | :heavy_check_mark: | `cosign`; the validator type must be set to `cosign`. |
 | `trustRoots[*].name` | - | :heavy_check_mark: | See [basics](../basics.md#validators). |
-| `trustRoots[*].key` | - | :heavy_check_mark: | See [basics](../basics.md#validators). Public key from `cosign.pub` file or [KMS URI](https://github.com/sigstore/cosign/blob/main/KMS.md). See additional notes [below](#kms-support). |
+| `trustRoots[*].key` | - | :heavy_check_mark: if not using keyless | See [basics](../basics.md#validators). Public key from `cosign.pub` file or [KMS URI](https://docs.sigstore.dev/key_management/overview/#basic-usage). See additional notes [below](#kms-support). |
+| `trustRoots[*].keyless.issuer` | - | :heavy_check_mark: if not using a key or issuerRegex | The OIDC provider name which attests the identity. |
+| `trustRoots[*].keyless.subject` | - | :heavy_check_mark: if not using a key or subjectRegex | The identity that created the keyless signature. Usually an email address. |
+| `trustRoots[*].keyless.issuerRegex` | - | :heavy_check_mark: if not using a key or issuer | Regex for the OIDC provider name which attests the identity. So, either the full URL of the OIDC provider can be used or only parts of it. |
+| `trustRoots[*].keyless.subjectRegex` | - | :heavy_check_mark: if not using a key or subject | Regex of the identity that created the keyless signature. Usually an email address. So, either the full email address can be used or only parts of it (e.g. `your.mail@example.org` or `your.mail`). |
 | `host.rekor` | `rekor.sigstore.dev` | - | Rekor URL to use for validation against the transparency log (default sigstore instance is `rekor.sigstore.dev`). Setting `host` enforces successful transparency log check to pass verification. See additional notes [below](#transparency-log-verification). |
+| `host.rekorPubkey` | Public key of `rekor.sigstore.dev` | - | Public key used to verify signature of log entries from Rekor. |
+| `host.fulcioCert` | Root and intermediate certificates belonging to `fulcio.sigstore.dev` | - | The root certificate belonging the Fulcio CA which was used to create keyless signatures. |
+| `host.ctLogPubkey` | Public keys for the certificate transparency log provided by Sigstore | - | The public keys needed for verifying Signed Certificate Timestamps (SCT). |
```

Comment:
Either this should be `host.ctLogPubkeys` and we should accept multiple keys or the docs should state that only a single public key can be provided, e.g. no key*s*